internal/pods/schedule.go

```diff
@@ -270,3 +285,35 @@ func addNodeSelectorTerm(nodeSelTerms []v1.NodeSelectorTerm, label string,
 		},
 	}})
 }
+
+func checkNSLabels(nsLabels map[string]string) {
+	expectedLabels := map[string]string{
+		"pod-security.kubernetes.io/enforce": "privileged",
+		"pod-security.kubernetes.io/audit":   "privileged",
+		"pod-security.kubernetes.io/warn":    "privileged",
+	}
+
+	missingLabels := map[string]string{}
+
+	for key := range expectedLabels {
+		val, found := nsLabels[key]
+		if !found || val != "privileged" {
+			missingLabels[key] = "privileged"
+		}
+	}
```

Comment:
Instead of hard-coding the expected values here, use those from the map:

```suggestion
	for key, expectedLabel := range expectedLabels {
		actualLabel, found := nsLabels[key]
		if !found || actualLabel != expectedLabel {
			missingLabels[key] = expectedLabel
		}
	}
```